aesara/link/numba/dispatch/basic.py

```diff
@@ -498,8 +499,6 @@ def numba_funcify_Subtensor(op, node, **kwargs):
 
 
 @numba_funcify.register(IncSubtensor)
-@numba_funcify.register(AdvancedIncSubtensor)
```

Comment:
What happened to `AdvancedIncSubtensor` (without the `1`)?

Reply:
I saw now you mention it in the PR description...

Reply:
done